presto-main/src/main/java/com/facebook/presto/operator/scalar/QuantileDigestFunctions.java

```diff
@@ -63,6 +67,36 @@ public static long valueAtQuantileBigint(@SqlType("qdigest(bigint)") Slice input
         return new QuantileDigest(input).getQuantile(quantile);
     }
 
+    @ScalarFunction("quantile_at_value")
+    @Description("Given an input x between min/max values of qdigest, find which quantile is represented by that value")
+    @SqlType(StandardTypes.DOUBLE)
+    public static double quantileAtValueDouble(@SqlType("qdigest(double)") Slice input, @SqlType(StandardTypes.DOUBLE) double value)
+    {
+        return quantileAtValueBigint(input, doubleToSortableLong(value));
+    }
+
+    @ScalarFunction("quantile_at_value")
+    @Description("Given an input x between min/max values of qdigest, find which quantile is represented by that value")
+    @SqlType(StandardTypes.DOUBLE)
+    public static double quantileAtValueReal(@SqlType("qdigest(real)") Slice input, @SqlType(StandardTypes.REAL) long value)
+    {
+        return quantileAtValueBigint(input, floatToSortableInt(intBitsToFloat((int) value)));
+    }
+
+    @ScalarFunction("quantile_at_value")
+    @Description("Given an input x between min/max values of qdigest, find which quantile is represented by that value")
+    @SqlType(StandardTypes.DOUBLE)
+    public static double quantileAtValueBigint(@SqlType("qdigest(bigint)") Slice input, @SqlType(StandardTypes.BIGINT) long value)
+    {
+        QuantileDigest digest = new QuantileDigest(input);
+        // to be consistent with non-empty scenarios. When no value in the digest is smaller than the input, always return 0.0
```

Comment:
- When no value in the non-empty qdigest is larger than the input,  we actually return 1. Thus, I recommend to remove this comment and just document the behavior is enough. Zero is chosen since it is more like empty.

- Throw an error for empty qdigest is also not ideal,  since value_at_quantile function doesn't throw error for empty qdigest.